pandas-stubs/_typing.pyi

```diff
@@ -160,6 +160,7 @@ AstypeArg: TypeAlias = (
     | TimedeltaDtypeArg
     | TimestampDtypeArg
     | CategoricalDtype
+    | CategoryDtypeArg
```

Comment:
Ideally, we should change `CategoricalDtypeArg` to be `CategoricalDtype | Literal["category"]`